### PR TITLE
Enable usage without kong

### PR DESCRIPTION
**Pull request by landorg**
We are using kong with it's ingress controller as our main LB. 
We'd like to disable the included kong and point our main kong to graphdb 

I didn't really get the reason why you are including kong in here. 
Could you please explain why it is there?

I see it adds the X-Request-Id header and it makes all instances available from the outside. I'm not sure if we want that. 
Is it enough to expose only the master to the outside or do we need to expose the workers too like you do with kong?

### CODE REVIEW

**Comment by nikolaykolev** [comment]
Kong is used for better management of the exposed services and on what URLs they should be exposed. Usually exposing only master-1 is enough, but some users might want to use their other instances in some other ways, for example a worker instance might hold a standalone repository not connected to a cluster or for example someone might want to use master-2 as a primary master for another repository's cluster. Kong provides some flexibility in unusual scenarios, it can be toggled off of course.

Your suggestion here is great btw! Thanks for the contributions, also if you think the chart lacks any features please open an issue and we can discuss. Or write to me directly at nikolay.kolev@ontotext.com

**Comment by landorg** [comment]
Thank You.
That works for me so far.

File: templates/gateway/ingress.yaml [hunk 1]
```diff
@@ -28,7 +28,13 @@ spec:
             pathType: Prefix
             backend:
               service:
+                {{ if .Values.kong.enabled }}
                 name: kong-proxy
                 port:
                   number: 8000
+                {{ else }}
+                name: graphdb-master-1
```
[the remark this answers is not in the document]
Comment (landorg) [reply templates/gateway/ingress.yaml:36]: hi and sorry for the delay. 
good question. 
How is this handled in your kong setup? 
We are currently using 1m_3w and in general we only access the master.
I guess we can't use a kubernetes lb across all masters here.

**Comment by yaskoo** [comment]
Kong hasn't been part of the chart since GraphDB 10. I'm closing this as stale.

**Pull request closed without merging**